newrelic/api/graphql_trace.py

```diff
@@ -36,9 +36,9 @@ def __repr__(self):
         return '<%s %s>' % (self.__class__.__name__, dict())
 
     def finalize_data(self, *args, **kwargs):
-        self._add_agent_attribute("graphql.operation.type", self.operation_type)
-        self._add_agent_attribute("graphql.operation.name", self.operation_name)
-        self._add_agent_attribute("graphql.operation.deepestPath", self.deepest_path)
+        self._add_agent_attribute("graphql.operation.type", self.operation_type or "<unknown>")
+        self._add_agent_attribute("graphql.operation.name", self.operation_name or "<anonymous>")
+        self._add_agent_attribute("graphql.operation.deepestPath", self.deepest_path or "<unknown>")
```

Comment:
Going to change this in a future PR, we should just set this once and be done with it instead of tracking down everywhere it gets used and reevaluating. This is fine for now though.